### PR TITLE
Add inline :code: tags

### PR DESCRIPTION
**Pull request by jmsfwk**

### CODE REVIEW

**Comment by jonashaag** [comment]
Thanks for the patch! I couldn't find any documentation about the `:code:` markup. How is it different from using two backticks (``` ``foo``` ``)?

**Comment by jmsfwk** [comment]
Apparently two backticks is an inline literal whereas `:code:` is a Interpreted Text Role.

As far as I can tell the double backticks only render the content in a monospace font, but `:code:` creates a `<code>` element.

http://stackoverflow.com/a/12365251/3935089

File: docs/views.rst [hunk 1]
```diff
@@ -31,10 +31,10 @@ A simple formset::
             # do stuff
             return super(MyFormSetView, self).formset_valid(formset)
 
-This view will render the template `myformset.html` with a context variable
-`formset` representing the formset of MyForm. Once POSTed and successfully
-validated, `formset_valid` will be called which is where your handling logic
-goes, then it redirects to `success_url`.
+This view will render the template :code:`myformset.html` with a context variable
```
Comment (jonashaag) [inline docs/views.rst]: Use ``:file:`myformset.html``` here

**Comment by jonashaag** [comment]
Ok then it seems like everyone has been doing this wrong forever, because ``` `` ``` is *very* widespread whereas I have seen almost no use of `:code:`.

**Pull request merged**